### DISCUSSION TITLE
No signature of method: java.util.LinkedHashMap.isDefined() is applicable for argument types: (String) values: [analytics_auto_collection_enabled]

### DISCUSSION THREAD

**Discussion by Shahrukh0396** [invertase/react-native-firebase#5818] [Q&A] [answered]
No signature of method: java.util.LinkedHashMap.isDefined() is applicable for argument types: (String) values: [analytics_auto_collection_enabled
![Screenshot 2021-10-28 at 1 01 25 PM](https://user-images.githubusercontent.com/33844214/139213097-8a4c2c5d-3629-4c24-8f2f-a25af3a97c53.png)
].

**Comment by mikehardy** [comment, marked as answer]
Hi there, this in issue report not a discussion, but a few points of order: images are terrible for troubleshooting, basically don't ever post images for troubleshooting as they are not quotable and don't show up well for people on mobile devices. Use triple-backticks and paste the text in between sets of triple backticks like this:

```
my error message here
```

Second, if you have an issue report, post it as an issue and *follow the issue template* as that requests all the information we would need to make guesses to help you, but you've provided none of that, so no guesses are possible, except that likely you are just using out of date gradle / android gradle plugin.

You can see analytics added to a project and compiling successfully by following the steps here (or just running it) https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh so we know it works, thus this will be a local configuration issue and you'll need to examine what's different between our integration/build demonstrator and your project to find the answer - that really should help, hope it does
